static/bidder-params/adhese.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Adhese Adapter Parameters",
+	"description": "Validation for parameters handled by the Adhese adapter",
+	"type": "object",
+	"properties": {
+		"account": {
+			"type": "string",
+			"description": "Your Adhese account name. If unknown, please contact your sales rep"
```

Comment:
Please consider adding a `/doc/bidders/adhese.md` with instructions on how to contact an Adhese salea rep to register for these values.